lib/add_bc_user.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+echo "This script add's a user with password to the bridghead"
+read -p 'Username: ' bc_user
+read -sp 'Password: ' bc_password
+
+echo 
+
+bc=$(docker run --rm -ti xmartlabs/htpasswd $bc_user $bc_password)
+
+if [ -z $bc_auth_users ]; then
+    printf "Please run: export bc_auth_users=\"%q\"" $bc
```

Comment:
Please move this to /etc/systemd/system/bridgehead@\.service.d/xxx.conf
Otherwise the password and user would be lost after system restart.